From 99710017843636624dfc15bb42a8d0dea41c927d Mon Sep 17 00:00:00 2001
From: Tsz-wo Sze <szetszwo@apache.org>
Date: Mon, 24 Mar 2014 18:38:17 +0000
Subject: [PATCH 1276/2748] svn merge -c 1580977 from trunk for HADOOP-10015.
 UserGroupInformation prints out excessive
 warnings.

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-2@1580978 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 77b05150a79cb02300f90266bf1e397b11eb4499)

Change-Id: Id5038a4401494a2d7a545c8359aa90baac69b5bb
---
 .../hadoop/security/UserGroupInformation.java      |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java
index c7c52e8..86ab1d6 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java
@@ -1693,7 +1693,9 @@ protected Subject getSubject() {
       return Subject.doAs(subject, action);
     } catch (PrivilegedActionException pae) {
       Throwable cause = pae.getCause();
-      LOG.warn("PriviledgedActionException as:"+this+" cause:"+cause);
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("PrivilegedActionException as:" + this + " cause:" + cause);
+      }
       if (cause instanceof IOException) {
         throw (IOException) cause;
       } else if (cause instanceof Error) {
-- 
1.7.9.5

